test-suite/TestSpec.hs

```diff
@@ -19,6 +19,14 @@ import Text.Layout.Table.Primitives.AlignInfo
 import Text.Layout.Table.Justify
 import Text.Layout.Table.Cell.Formatted
 
+
+newtype NonControlASCIIString = NonControlASCIIString String
+  deriving (Eq, Ord, Show)
+
+instance Arbitrary NonControlASCIIString where
+  arbitrary = NonControlASCIIString <$> listOf (chooseEnum ('\32', '\126'))
```

Comment:
Please put a small comment there which explains it and points to a specification or Wikipedia page.

Reply:
Done.